bc_obps/reporting/migrations/0086_update_storage_of_petroleum_products_shcema.py

```diff
@@ -0,0 +1,48 @@
+# Generated by Django 5.2 on 2025-04-16 18:32
+
+from django.db import migrations
+import os
+import json
+
+
+def update_storage_of_petroleum_products(apps, schema_editor):
+    '''
+    Add initial data to reporting.activitysourcetypejsonschema
+    '''
+
+    # Load JSON schema
+    cwd = os.getcwd()
+    schema_path = os.path.join(
+        cwd, 'reporting/json_schemas/2024/storage_of_petroleum_products/storage_of_petroleum_products.json'
```

Comment:
We could use this PR to rename the schema to follow the convention "activity/source_type.json" like all the others: `above_ground_storage_tanks.json`